docs/resources/apikey.md

```diff
@@ -0,0 +1,41 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "streamnative_apikey Resource - terraform-provider-streamnative"
+subcategory: ""
+description: |-
+  
+---
+
+# streamnative_apikey (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `instance_name` (String) The pulsar instance name
+- `name` (String) The name of the api key
+- `organization` (String) The organization name
+- `service_account_name` (String) The service account name
+
+### Optional
+
+- `description` (String)
+- `expiration_time` (String) The expiration time of the api key, you can set it to 1m(one minute), 1h(one hour), 1d(one day) or this time format 03/08/2024 15:30:00, if you set it 'No Expiration Time', it will never expire, if you don't set it, it will be set to 30d(30 days) by default
```

Comment:
Use `0` to be better than `No Expiration Time` to represent never expire

Reply:
Just see on the console it uses `No Expiration Time` for never expire

Reply:
NO, it's just the description on UI, in fact, the param value is 0

<img width="414" alt="image" src="https://github.com/streamnative/terraform-provider-streamnative/assets/76024046/617ce5f3-493b-43cf-a67a-52a923a47016">

Reply:
and when you use terminal, input `0` is much easier than a long string `No Expiration Time`, wdyt?

Reply:
Fixed